mailbox/api/src/main/java/org/apache/james/mailbox/model/MailboxQuery.java

```diff
@@ -33,62 +33,65 @@
  * Expresses select criteria for mailboxes.
  */
 public final class MailboxQuery {
-
-    private final MailboxPath base;
-
-    private final String expression;
-
-    private final char pathDelimiter;
-    
-    private final Pattern pattern;
+    public static final char SQL_WILDCARD_CHAR = '%';
```

Comment:
Sorry but I really do not understand this constant in what it is different from ``LOCALWILDCARD``. What does SQL have to do with MailboxQuery isn't this class supposed to be backend agnostic ?